Tests/AblyChatTests/DefaultConnectionTests.swift

```diff
@@ -0,0 +1,196 @@
+import Ably
+@testable import AblyChat
+import Testing
+
+@MainActor
+struct DefaultConnectionTests {
+    // MARK: - CHA-CS1: Connection Status Values
+
+    // @spec CHA-CS1a
```

Comment:
These spec references are generating warnings about non-testable spec points in the spec coverage report. Isn't CHA-CS3 the relevant spec point here?